config/flags/flags.go

```diff
@@ -250,10 +251,11 @@ func NewProvider(flagset FlagSet, fallback config.Provider) *Provider {
 		config.DefaultAuthCacheDisabled,
 		"disable the CLI's caching of authentication sessions",
 	)
-	flagset.StringArray(
```

Comment:
Does this affect the way users currently input a list of endpoints? 

I'm happy adding new ways, but I don't want to break the existing workflow at the same time

Reply:
I don't think the chance of someone currently using scripts/automation together with this CLI tool is high enough to make it worth the added complexity of implementing two inputs for the same thing and logic to merge both

Reply:
Let's ask Ferran, as I know some customers have automation around us